gateway/src/main/java/com/hurence/webapiservice/http/grafana/GrafanaApi.java

```diff
@@ -16,6 +16,9 @@ default Router getGraphanaRouter(Vertx vertx) {
         router.post("/query")
                 .produces("application/json")
                 .handler(this::query);
+        router.post("/export/csv")
+                .produces("text/plain")
```

Comment:
why text/plain ? Are you sur this fits better than test/csv as asked in the issue ?

Reply:
done